src/nwb_conversion_tools/tools/data_transfers.py

```diff
@@ -8,9 +8,14 @@
 from warnings import warn
 from shutil import rmtree
 from time import sleep, time
+from tempfile import mkdtemp
 
 import psutil
 from tqdm import tqdm
+from pynwb import NWBHDF5IO
+from dandi.download import download
+from dandi.organize import organize
+from dandi.upload import upload
 
```

Comment:
Dandi chose rather generic names for their functions. As this whole module is for data transfer maybe we can alias them (i.e `import as dandi_download`)?

Reply:
Good idea, added